README.md

```diff
@@ -43,14 +44,27 @@ Install additional dependencies:
 
 ### Ubuntu
 
-Gazebo Garden:
+Recommended - Use rosdep with
```

Comment:
The problem is that this assumes a ROS install, and it should be possible to use this plugin without ROS, so recommending rosdep adds a dependency that was not previously present.

Reply:
Note this PR is targetted at the `ros2` branch. What if I change to `Installation using rosdep:` and let users decide which method is appropriate?

Reply:
> Note this PR is targetted at the ros2 branch. What if I change to Installation using rosdep: and let users decide which method is appropriate?

This PR should target `main`, then forward port the doc changes to `ros2` in a separate PR.

The `ros2` branch should only differ from `main` by the changes to the resource prefix in the models that are used in `rviz2`. This is the `model://` vs `package://{model_name}/...` issue caused by `sdformat_urdf`. The aim is that this split goes away when the issue is solved upstream.

Reply:
Got it. Many thanks for the clarification. The changes are complete.